02-setup.md

```diff
@@ -48,9 +49,17 @@ He also has to set his favorite text editor, following this table:
 | Notepad++ (Win)    | `$ git config --global core.editor "'c:/program files (x86)/Notepad++/notepad++.exe' -multiInst -notabbar -nosession -noPlugin"`|
 | Kate (Linux)       | `$ git config --global core.editor "kate"`       |
 | Gedit (Linux)      | `$ git config --global core.editor "gedit -s -w"`   |
+| Scratch (Linux)       | `$ git config --global core.editor "scratch-text-editor"`  |
 | emacs              | `$ git config --global core.editor "emacs"`   |
+
+It is also possible to configure "Vim" as the text editor for Git if you want to change it back. Note that Vim is the default editor for Git so this is only needed if you have changed it already.
+
+| Default Editor             | Configuration command                            |
+|:-------------------|:-------------------------------------------------|
 | vim                | `$ git config --global core.editor "vim"`   |
 
+If you haven't used Vim before and wish to exit a session, type <kbd>Esc</kbd> then <kbd>:q!</kbd> and <kbd>Enter</kbd>.
```

Comment:
This is a very, very welcomed addition! Could you make it a callout? `vim` is not only the default for `git`, but for some other programs, so this is useful to have stressed.

Also, we prefer to avoid the html in markdown, so it'd be great if you used just back ticks.